File: SEP3 - TIER 2/src/main/java/com/example/tier3NetworkPackages/NetworkPackage.java
package com.example.tier3NetworkPackages;

public class NetworkPackage
{
  private NetworkType type;
  private long id;

  public NetworkPackage(NetworkType type, Long id)
  {
    this.type = type;
    this.id = id;
  }

  public NetworkType getType()
  {
    return type;
  }

  public long getId()
  {
    return id;
  }

  public void setId(long id)
  {
    this.id = id;
  }
}

File: SEP3 - TIER1/Gradebook/NetworkPackages/NetworkPackage.cs
namespace WebApplication.NetworkPackages
{
    public class NetworkPackage
    {
        private NetworkType _networkType;
        private long _id;

        public NetworkPackage(NetworkType networkType, long id)
        {
            this._networkType = networkType;
            this._id = id;
        }

        public NetworkType GetType()
        {
            return _networkType;
            
        }
        
        public long GetId()
        {
            return _id;
        }

        public void SetId(long id)
        {
            this._id = id;
        }
    }
}